.infra/files/scripts/anonymise-users.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+/opt/app/tools/docker-compose.sh run --rm --no-deps server yarn cli anonymiseUsers
```

Comment:
```suggestion
/opt/app/tools/docker-compose.sh run --rm --no-deps server yarn cli anonymizeUsers
```

Pour harmoniser les `anonymise` vs `anonymize`